masq/src/commands/setup_command.rs

```diff
@@ -338,4 +371,59 @@ ERRORS:
 ip                            No sir, I don't like it.\n\
 \n");
     }
+
+    #[test]
+    fn setup_command_with_data_directory_shows_right_path() {
+        vec![
+            (None, None, Some("polygon-mainnet"), Some("/home/cooga/.local/MASQ/polygon-mainnet"), Some(None), UiSetupResponseValueStatus::Default, UiSetupResponseValueStatus::Default),
+            (Some("polygon-mumbai"), None, Some("polygon-mumbai"), Some("/home/cooga/.local/MASQ/polygon-mumbai"), Some(None), UiSetupResponseValueStatus::Default, UiSetupResponseValueStatus::Default),
```

Comment:
`status_chain` should be `UiSetupResponseValueStatus::Set`, not `UiSetupResponseValueStatus::Default`.